doc/jsk_pcl_ros/nodes/organized_pass_through.rst

```diff
@@ -23,7 +23,7 @@ Publishing Topic
 
 -  ``~output`` (``sensor_msgs/PointCloud2``)
 
-  Filtered point cloud.
+  Filtered point cloud where ``NaN`` points are removed.
```

Comment:
Ugh, sorry. possibly I was wrong, removing nan points always with `keep_organized:=false` causes problems in below cases:

**CASE1**

keep_organized:=false
- Pointcloud1 (height=1, width=1000) min_index: 500, max_index: 750
- Pointcloud2 (height=1, width=1000) min_index: 500, max_index: 750

**CASE2**

keep_organized:=false
- Pointcloud1 (height=480, width=640) min_index: 500, max_index: 750
- Pointcloud2 (height=480, width=640) min_index: 500, max_index: 750


In this case, user assumes the output pointcloud has same size, but if we always remove nan points, the size will be changed.